README.md

```diff
@@ -0,0 +1,9 @@
+# Snowball
+
+This contains the source code for the Snowball compiler and the stemming algorithms on the website.
+
+See http://snowballstem.org/ for more details.
+
+## Compile
+
+`make -f GNUmakefile dist`
```

Comment:
GNU make will automatically use `GNUmakefile`; Non-GNU make is unlikely to cope with the use of GNU extensions.

If it actually works with other make implementations, we should just rename it to `Makefile`.